### DISCUSSION TITLE
File Storage Setup - AWS S3 Adapter

### DISCUSSION THREAD

**Discussion by ghost** [pimcore/pimcore#9190] [Q&A / Support] [answered]
Hello, 

I am new to the topic and have succesfully installed Pimcore/Skeleton on the server.  

![image](https://user-images.githubusercontent.com/68632142/119146674-27530d80-ba4b-11eb-8d5d-c87986a37f92.png)

 
In order to upload Pimcore assets to my AWS S3 bucket I followed the docs:
https://pimcore.com/docs/pimcore/current/Development_Documentation/Installation_and_Upgrade/System_Setup_and_Hosting/File_Storage_Setup.html

This means, so far I kept the default configuration file in its original state: vendor/pimcore/pimcore/bundles/CoreBundle/Resources/config/pimcore/flysystem.yml

 
![image](https://user-images.githubusercontent.com/68632142/119146733-376aed00-ba4b-11eb-8317-35362b840dd2.png)

But I customized my AWS S3 Adapter for assets in a new file: config/packages/prod/flysystem.yaml
As far as I understand these specific configuration entries will override the default entries during runtime. I can upload assets manually to my S3 bucket from aws cli. But when I log in to pimcore frontend and create new folders and upload new assets, these are stored locally. Nothing is uploaded to my S3 bucket.  Any help would be greatly appreciated!

**Comment by fashxp** [comment]
Hi,
did you verify that your configuration is loaded? Looks like Pimcore is not loading the config file - maybe not included or container rebuild necessary?

**Comment by ghost** [comment, marked as answer]
see https://github.com/pimcore/pimcore/discussions/9293#discussion-3391155
